xbmc/platform/darwin/ios/XBMCApplication.mm

```diff
@@ -197,7 +189,7 @@ static void XBMCsendEvent(id _self, SEL _cmd, UIEvent *event)
         int idx = GSEVENTKEY_KEYCODE;
```

Comment:
This value will never be used. I would remove `GSEVENTKEY_KEYCODE_IOS7` define, update the value of `GSEVENTKEY_KEYCODE` and remove the else below.